bugsnag-plugin-android-ndk/src/test/cpp/test_bsg_event.c

```diff
@@ -2,10 +2,23 @@
 #include <event.h>
 #include <utils/string.h>
 #include "../../main/assets/include/bugsnag.h"
+#include <event.h>
 
 bugsnag_event *init_event() {
     bugsnag_event *event = calloc(1, sizeof(bugsnag_event));
     bsg_strncpy_safe(event->context, "Foo", sizeof(event->context));
+
+    event->device.jailbroken = true;
+    event->device.total_memory = 1095092340;
+    bsg_strncpy_safe(event->device.id, "my-id-123", sizeof(event->device.id));
```

Comment:
Missing jailbroken, cpu_abi and total memory (free disk and free memory too, if we add them)

Reply:
`jailbroken` and `totalMemory` appear to be populated in `init_event()` and there are tests for both of them, is something else missing?

`cpuAbi` will be addressed in a separate PR as its a struct and will require a bit of thought (mentioned elsewhere) and we don't collect free disk/memory as it's not possible from a signal handler.

Reply:
Apologies - it's out of horizontal alignment as it's not a string, so I missed that!